### PR TITLE
Add a makeshift welding blanket

### LINKED ISSUE #62093: More ways to get welding blanket
### Is your feature request related to a problem? Please describe.

More and more weapons, armors ond other items are getting different steel grades, and best ones require very specific item in order to repair, an asbestos welding blanket. This is very rare item, especially in older worlds.

### Solution you would like.

1. Make refugee center trader sell it from time to time as his stock is already tool-oriented. This would help older worlds solve problem of lack of blankets. 
2. Way to craft asbestos blanket from asbestos. Then make asbestos sheets spawn in the world in some stores, at junkyards etc. 
Maybe some buildings should spawn asbestos when their roofs are destroyed or deconstructed. 

### Describe alternatives you have considered.

1. Add fiberglass welding blankets. Add fiberglass sheets. 
2. A way to mine and refine raw asbestos from earth. Probably out of scope of the game. 

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by ccaviness**
#### Summary
Content "Add a makeshift welding blanket"

#### Purpose of change
The welding kit requires a welding blanket, but welding blankets are quite rare and no recipe exists to create one. Doing it properly would require working with fiberglass or asbestos.

#### Describe the solution
Add a makeshift version made from fiber insulation batts from deconstructed ovens, which should be sufficient for welding in the apocalypse.

#### Describe alternatives you've considered
Damp towels?

Implementing fiberglass crafting.

#### Testing
Loaded in a test save. Constructed a makeshift welding blanket, and used it t construct a welding kit.

#### Additional context
Wild guesses made as to material quantities required. Suggestions on further refinement or added proficiences etc. requested. I don't have any real-life experience with welding, but I did come across [this Reddit post](https://www.reddit.com/r/Welding/comments/ususa2/good_fiberglass_blanket_alternative/) and [this other random forum post](https://www.v8buick.com/index.php?threads/what-do-you-use-as-a-welding-blanket.230909/) where alternatives to welding blankets are discussed. I don't think canvas+Nomex would pass government safety standards, but then we're welding in the apocalypse so I don't think that matters so much.

resolves #62093

### CODE REVIEW

**Comment by Drew4484** [comment]
I do want to add other welding blankets but Nomex isn't appropriate for this as the temperature is too high. Tempered welds need to be insulated at high heat for a long period of time, Nomex tops out around 370°c but the weld can be significantly hotter than that.

**Comment by MNG-cataclysm** [comment]
“Category” should be changed to “Content”, and the line where it currently is be a short description of the PR. This is to make the PR Validator happy and have this properly appear in changelogs.

**Comment by ccaviness** [comment]
> I do want to add other welding blankets but Nomex isn't appropriate for this as the temperature is too high. Tempered welds need to be insulated at high heat for a long period of time, Nomex tops out around 370°c but the weld can be significantly hotter than that.

Well, like I said, I've no experience with welding so I was just making guesses. Do you have any suggestions for craftable materials to use to create a makeshift welding blanket that would be good enough for the cataclysm?

> “Category” should be changed to “Content”, and the line where it currently is be a short description of the PR. This is to make the PR Validator happy and have this properly appear in changelogs.

Done.

**Comment by Drew4484** [comment]
> > I do want to add other welding blankets but Nomex isn't appropriate for this as the temperature is too high. Tempered welds need to be insulated at high heat for a long period of time, Nomex tops out around 370°c but the weld can be significantly hotter than that.
> 
> Well, like I said, I've no experience with welding so I was just making guesses. Do you have any suggestions for craftable materials to use to create a makeshift welding blanket that would be good enough for the cataclysm?
> 
> > “Category” should be changed to “Content”, and the line where it currently is be a short description of the PR. This is to make the PR Validator happy and have this properly appear in changelogs.
> 
> Done.

Fiberglass blankets are usually good for around 50 PWHTs which is good enough for our purposes. However adding fiberglass is a challenge as there's several materials available in different formats, so it will result in multiple materials being added.

**Comment by ccaviness** [comment]
Can you think of any way to build a workable welding blanket using the existing materials available in the game? Wool, cotton, canvas, metal, plastic, Kevlar, etc.?

**Comment by Zireael07** [comment]
Some ideas from here: https://www.weldpundit.com/what-is-a-welding-blanket-a-helpful-guide/

Leather, felt, ceramic seem to be options

**Comment by Drew4484** [comment]
> Some ideas from here: https://www.weldpundit.com/what-is-a-welding-blanket-a-helpful-guide/
> 
> Leather, felt, ceramic seem to be options

Some of these are better for catching sparks and globules of molten metal than as an actual insulator. Felt and leather aren't long-term durable, though leather lasts longer and would be relatively easy for a survivor to make. Fiberglass is the ideal, but the hardest to add.

**Comment by Zireael07** [comment]
@Drew4484 I doubt a survivor would be able to craft fiberglass though

**Comment by Drew4484** [comment]
> @Drew4484 I doubt a survivor would be able to craft fiberglass though

Weave, no, and new fibers no. But it's easy to craft with, and damaged weave can be reused as shake for patchwork repairs.

**Comment by adamkad1** [comment]
> 

This makes sense

**Comment by adamkad1** [comment]
> I can't help but think that [fibre insulation batts](https://nornagon.github.io/cdda-guide/#/item/rock_wool_bat) from oven disassembly were made exactly for this purpose. The item ID implies that it's [rock wool](https://en.wikipedia.org/wiki/Mineral_wool), which enjoys 700°C temperature resistance.
> 
> Caveat: I've never welded or experimented with the insulation from a dismantled oven before.

This makes sense

**Comment by ccaviness** [comment]
Ok, I re-did this to use a small pile of fiber insulation batts from deconstructed ovens instead. Also changed to use `sewing_aramids` which would use heavy-duty, Nomex, or Kevlar thread.

File: data/json/recipes/tools/tool.json [hunk 1]
```diff
@@ -1591,5 +1591,20 @@
...
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "tailor",
+    "difficulty": 2,
+    "time": "120 m",
+    "autolearn": [ [ "tailor", 3 ] ],
+    "book_learn": [ [ "welding_book", 1 ] ],
+    "using": [ [ "sewing_aramids", 50 ], [ "adhesive", 4 ] ],
+    "proficiencies": [ { "proficiency": "prof_polymerworking", "required": false, "time_multiplier": 1.5 } ],
+    "components": [ [ [ "rock_wool_bat", 15 ] ] ]
```
Comment (bombasticSlacks) [inline data/json/recipes/tools/tool.json:1608]:
```suggestion
    "components": [ [ [ "rock_wool_bat", 22 ] ] ]
```

This at least matches weights with some extra for scrap ends.

**Pull request merged**